aspnetcore/tutorials/min-web-api/samples/8.x/todo/Program.cs

```diff
@@ -92,14 +92,16 @@ await db.Todos.FindAsync(id)
 app.MapGet("/todoitems/complete", async (TodoDb db) =>
     await db.Todos.Where(t => t.IsComplete).ToListAsync());
 
+// <snippet_1a>
 app.MapGet("/todoitems/{id}", async (int id, TodoDb db) =>
     await db.Todos.FindAsync(id)
         is Todo todo
             ? Results.Ok(todo)
             : Results.NotFound());
+// </snippet_1a>
 
 // <snippet_1b>
- app.MapGet("/todos/{id}", async Task<Results<Ok<Todo>, NotFound>> (int id, TodoDb db) =>
+ app.MapGet("/todoitems/{id}", async Task<Results<Ok<Todo>, NotFound>> (int id, TodoDb db) =>
```

Comment:
This would result in a route conflict build warning for  `/todoitems/{id}` since there are two of them now.  I think the original author worked around that problem for the sake of keeping it all in one place by commenting out the duplicate route snippet below.  You might have to do the same for this one.   (Same for version 7.0)

Reply:
Thanks for catching the build warning Wade.

I've updated both versions by adding a comment around the second code block.